wiki/en/Include-Server-Commands.md

```diff
@@ -1,19 +1,19 @@
-- `-e`  or `--directoryserver`  Register the Server on a Directory (e.g. to set its genre (see also `-o`)). See [Server Types](#server-types) for further information.
+- `-d` or `--discononquit` Disconnect all Clients on quit. Normally, when a Server is stopped or restarted, any Clients that have not used their "Disconnect" buttons will re-establish connection when the Server comes back up again. Using this option forces Clients to manually re-establish their connections to the Server.
+- `-e` or `--directoryserver` Register the Server on a Directory (e.g. to set its genre (see also `-o`)). See [Server Types](#server-types) for further information.
 - `--directoryfile` Remember registered Servers even if the Directory is restarted. Directory Servers only. See [this guide](Directories) for further information.
-- `-f`  or `--listfilter`     Whitelist Servers registering on the Server list, format `ip address 1[;ip address 2]` Directories only.
--  `-F`  or `--fastupdate`     Reduces latency if Clients connect with "Enable Small Network Buffers" option. Requires faster CPU to avoid dropouts, and more bandwidth to enabled Clients.
--  `-l`  or `--log`            Enable logging, set path and file name                                                                    
-- `-L`  or `--licence`        Show an agreement window before users can connect
-- `-m`  or `--htmlstatus`     Enable HTML status file, set path and file name
-- `-o`  or `--serverinfo`     Location details in the format:  `[name];[city];[locale value]` (see [values](https://doc.qt.io/qt-5/qlocale.html#Country-enum)) Registered Servers only
-- `-P`  or `--delaypan`       Start with delay panning enabled See [notes](#delay-panning)
-- `-R`  or `--recording`      Set server recording directory; Server will record when a session is active by default. See [Options](#options).  
-- `--norecord`                Set server not to record by default (when recording is configured e.g via `-R`)
+- `-f` or `--listfilter` Whitelist Servers registering on the Server list, format `ip address 1[;ip address 2]` Directories only.
+- `-F` or `--fastupdate` Reduces latency if Clients connect with "Enable Small Network Buffers" option. Requires faster CPU to avoid dropouts, and more bandwidth to enabled Clients.
+- `-l` or `--log` Enable logging, set path and file name
+- `-L` or `--licence` Show an agreement window before users can connect
+- `-m` or `--htmlstatus` Enable HTML status file, set path and file name
+- `-o` or `--serverinfo` Location details in the format: `[name];[city];[country as two-letter ISO country code or Qt5 Locale]` (see [two-letter ISO country codes](http://statoids.com/wab.html) ("A-2" column) or [Qt5 Locale values](https://doc.qt.io/qt-5/qlocale.html#Country-enum)) Registered Servers only
```

Comment:
Thanks!

Could we also use Wikipedia as a more neutral and maybe persistent source here?

```suggestion
- `-o` or `--serverinfo` Location details in the format: `[name];[city];[country as two-letter ISO country code or Qt5 Locale]` (see [two-letter ISO country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Current_codes) or [Qt5 Locale values](https://doc.qt.io/qt-5/qlocale.html#Country-enum)) Registered Servers only
```

Reply:
You mean https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements ?

Reply:
> You mean https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements ? Yes, that'd be better.

Yeah, that section is better than my suggestion. :)